app/views/layouts/application.html.erb

```diff
@@ -9,7 +9,8 @@
 <body>
   
   <nav class="top-bar">
-    <h1><a href="/">SciFiReviewSite</a></h1>
+    <h1><a href="/">Quantum Complainers</a></h1>
+    <h3><a href="/">A Sci-Fi Movie Review Site</a></h3>
```

Comment:
These anchor tags could be updated to `link_to`'s since this is an erb file. They are also both linking to the root route?